### DISCUSSION TITLE
Feature: display text in title sequences

### DISCUSSION THREAD

**Discussion by spacek531** [OpenRCT2/OpenRCT2#15699] [Feature requests]
This is something I thought of while discussing map credit over at NE: the ability to display text below the menu buttons during title sequences. One message per map will be fine, but if the text can be sequenced like the other commands, that would be even better.

**Comment by janisozaur** [comment]
This would be nice if we ever do https://github.com/OpenRCT2/OpenRCT2/issues/2408, so we can display park and author name
